JenkinsfileStageMerge

```diff
@@ -0,0 +1,87 @@
+// Openshift project
+openshiftProject = "continuous-infra"
+DOCKER_REPO_URL = '172.30.254.79:5000'
+STABLE_LABEL = "stable"
+
+// Defaults for SCM operations
+env.ghprbGhRepository = env.ghprbGhRepository ?: 'CentOS-PaaS-SIG/ci-pipeline'
```

Comment:
This paired with line 14 should be changed I think. Nothing in here seems to use packagepipelineUtils, so it looks like we only need the checkout of the ci-pipeline, which is what is currently taking place. But, the ghprb variables will be upstream-fedora-pipeline repo relevant, so don't make sense here. I think you can just checkout master of ci-pipeline simply like is done in the Jenkinsfile and stuff